Content.Shared/Doors/DoorEvents.cs

```diff
@@ -27,13 +24,6 @@ public class BeforeDoorOpenedEvent : CancellableEntityEventArgs
     {
     }
 
-    /// <summary>
-    /// Raised when the door is successfully opened.
-    /// </summary>
-    public class OnDoorOpenedEvent : HandledEntityEventArgs
-    {
-    }
```

Comment:
This event, along with `OnDoorClosedEvent` and `OnDoorDeniedEvent` were defined, but never raised or used anywhere. In order to avoid confusion I've just removed them.